Cargo.toml

```
@@ -5,20 +5,26 @@ name = "twilight-http-proxy"
 version = "0.1.0"
 
 [dependencies]
-twilight-http = { version = "0.5", default-features = false, features = ["rustls-webpki-roots"] }
-hyper = { version = "0.14", features = ["tcp", "server", "http1", "http2"] }
+dashmap = "4.0"
 http = "0.2"
+hyper = { version = "0.14", features = ["tcp", "server", "http1", "http2"] }
+tokio = { version = "1.0", features = ["rt", "rt-multi-thread", "macros"] }
```

"rt-multi-thread" implies "rt"
```suggestion
tokio = { version = "1.0", features = ["rt-multi-thread", "macros"] }
```